lib/test.js

```diff
@@ -2,6 +2,14 @@
 var isGenerator = require('is-generator').fn;
 var Promise = require('bluebird');
 var setImmediate = require('set-immediate-shim');
+var powerAssert = require('power-assert').customize({
+	output: {
+		renderers: [
+			require('power-assert-renderers').AssertionRenderer,
+			require('power-assert-renderers').SuccinctRenderer
+		]
+	}
+});
```

Comment:
Why can't this be done at require time in https://github.com/sindresorhus/ava/pull/46/files#diff-c56ba0fef1bb510acefb4680eb31e9c6R21?